GeosCore/photolysis_mod.F90

```diff
@@ -958,15 +958,19 @@ SUBROUTINE RD_AOD( Input_Opt, State_Chm, RC )
 
        ! Test if the file exists and define an output string
        IF ( FileExists ) THEN
-          FileMsg = 'FAST-JX (RD_AOD): Opening'
+          FileMsg = '(RD_AOD): Opening'
        ELSE
-          FileMsg = 'FAST-JX (RD_AOD): REQUIRED FILE NOT FOUND'
+          FileMsg = '(RD_AOD): REQUIRED FILE NOT FOUND'
        ENDIF
 
        ! Write to stdout for both regular and dry-run simulations
        IF ( Input_Opt%amIRoot ) THEN
-          WRITE( 6, 300 ) TRIM( FileMsg ), TRIM( ThisFile )
-300       FORMAT( a, ' ', a )
+#ifdef FASTJX
```

Comment:
Is variable "ThisFile" a full path? If yes I think we don't need to print "FAST-JX" and "CLOUD-J" since the path will be self-explanatory. Also, RD_AOD is always called regardless of whether using fast-jx or cloud-j, or even photolysis. The dat files, e.g. so4.dat, are not used in photolysis and happen to be stored in the fast-jx/cloud-j folders due to legacy.

Reply:
I can remove the `FAST-JX` and `CLOUD-J` so that it just says `RD_AOD`.

Reply:
Thanks!